### PR TITLE
Added auth and added scaffolding to run twice

### PR DESCRIPTION
**Pull request by GarethLewin**
@aasted

### CODE REVIEW

File: auth/auth.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
+/*
+Auth providor using Google+ and secure session cookies with gorilla
```
Comment (aasted) [inline auth/auth.go]: Hmm, looks like I misspelled provider

File: build/scripts/run_blueprint.sh [hunk 2]
```diff
@@ -11,7 +11,12 @@ export CONFIG_PREFIX="s3://$S3_CONFIG_BUCKET/$VPC_SUBNET_TAG/$CLOUD_APP/$CLOUD_E
...
 
-exec ./blueprint \
-  -scoopURL="${SCOOP_URL}" \
+exec ./blueprint "$@"                                        \
+  -scoopURL="${SCOOP_URL}"                                   \
+  -adminEmails="${ENG_ADMINS};${ANL_ADMINS}"                 \
+  -cookieSecret=${COOKIE_SECRET}                             \
+  -googleClientID=${GOOGLE_CLIENT_ID}                        \
+  -googleClientSecret=${GOOGLE_CLIENT_SECRET}                \
+  -loginURLRedirect=${LOGIN_URL_REDIRECT}					 \
```
Comment (aasted) [inline build/scripts/run_blueprint.sh]: Spacing?

**Comment by aasted** [comment]
LGTM

File: build/scripts/run_blueprint.sh (hunk 2 above)
Comment (GarethLewin) [inline build/scripts/run_blueprint.sh]: Not sure I follow? Too many spaces before the \s ?

File: auth/auth.go (hunk 1 above)
Comment (GarethLewin) [inline auth/auth.go]: Ok, when I refactor to using github we can fix that :)

**Comment by GarethLewin** [comment]
Updated, see http://jenkins-master-0.prod.us-west2.justin.tv/view/Science/job/science-blueprint/18/console to verify that the build worked

File: build/scripts/run_blueprint.sh (hunk 2 above)
Comment (aasted) [inline build/scripts/run_blueprint.sh]: Yep; there's just a random spacing before the \; looks like a tab error

Comment (aasted) [inline build/scripts/run_blueprint.sh]: Just on this line; the others are all identical

**Pull request closed without merging**